src/main/java/edu/project1/ConsoleHangman.java

```diff
@@ -0,0 +1,108 @@
+package edu.project1;
+
+import java.util.Scanner;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ConsoleHangman {
+
+    private final static Logger LOGGER = LogManager.getLogger();
+    static final String EXIT = "exit";
+    static final String FINISH = "finish";
+    static final String LOSE = "You lost!";
+
+    private Session session;
+
+    //Call this method to start the game
+    @SuppressWarnings("MagicNumber") public void playGame() {
+
+        Scanner sc = new Scanner(System.in);
+
+        String line = "";
+
+        LOGGER.info("Starting game...");
+
+        gameLoop:
+        while (!line.equals(EXIT)) {
+            LOGGER.info("Starting round...");
+            LOGGER.info("Input max possible mistakes");
+
+            int maxMistakes = 5;
+            maxMistakes = sc.nextInt();
+            sc.nextLine(); //idk why but this method returns empty line without reading an actual stdin
+
+            session = new Session(maxMistakes);
+            line = "";
+
+            roundLoop:
+            while (!line.equals(FINISH)) {
```

Comment:
`BUG` сделай 
```suggestion
            while (! FINISH.equals(line))) {
``` 
Так лучше писать, т.к. если даже line как-то окажется null, мы не упадём с NPE, да и вообще любые константы с переменной лучше так сравнивать